web/concrete/src/User/User.php

```diff
@@ -125,10 +127,10 @@ public function getUserInfoObject()
 
     public function __construct()
     {
-        $app = Application::getFacadeApplication();
+        $this->app = Application::getFacadeApplication();
```

Comment:
The reason I didn't do this is because the User class does not need need to be aware of the application at all, all of these individual functions needing awareness are examples of scope creep. Instead I opted for accessing the static property that should always be set on the `Application` facade, this should be just as - if not more - performant than what we have now.

In the future, these methods that need awareness of the application should be moved out into their own modules separate from the user instance that each individually have awareness of the application instance.